src/legacy/core_plugins/telemetry/server/collection_manager.ts

```diff
@@ -17,20 +17,72 @@
  * under the License.
  */
 
-class TelemetryCollectionManager {
-  private getterMethod?: any;
+import { encryptTelemetry } from './collectors';
+
+export type EncryptedStatsGetterConfig = { unencrypted: false } & {
+  server: any;
+  start: any;
```

Comment:
I think `start` and `end` are ISO strings, perhaps nullable.  Could we type them that way, instead of `any`?